Chapter-3-microservice-extraction/Fitnet/Src/Fitnet/appsettings.json

```diff
@@ -1,7 +1,7 @@
 {
   "Logging": {
     "LogLevel": {
-      "Default": "Information",
+      "Default": "Debug",
```

Comment:
Why debug in appsettings? Appsettings.json is considered as production ready configuration (even if not used) and thats why Logging level is set to information

Reply:
Yeah, Fixed